packages/fuel-gauge/src/contract-factory.test.ts

```diff
@@ -245,21 +253,146 @@ describe('Contract Factory', () => {
     );
   });
 
-  it('should not deploy contracts greater than 100KB', async () => {
+  it('should not deploy large contracts via create', async () => {
     using launched = await launchTestNode();
     const {
       wallets: [wallet],
     } = launched;
 
-    const largeByteCode = `0x${'00'.repeat(112400)}`;
-    const factory = new ContractFactory(largeByteCode, StorageTestContract.abi, wallet);
+    const factory = new ContractFactory(LargeContractFactory.bytecode, LargeContract.abi, wallet);
 
     await expectToThrowFuelError(
-      async () => factory.deploy(),
+      () => factory.deployContract(),
       new FuelError(
         ErrorCode.CONTRACT_SIZE_EXCEEDS_LIMIT,
         'Contract bytecode is too large. Max contract size is 100KB'
       )
     );
   });
+
+  it('deploys large contracts via blobs [byte aligned]', async () => {
+    using launched = await launchTestNode({
+      providerOptions: {
+        cacheUtxo: -1,
+      },
+    });
+
+    const {
+      wallets: [wallet],
+    } = launched;
+    const factory = new ContractFactory(LargeContractFactory.bytecode, LargeContract.abi, wallet);
```

Comment:
Now you can use typegen factory classes directly:

```suggestion
    const factory = new LargeContractFactory(wallet);
```

Reply:
The issue here is the typegen factory only exposes the A|B deploy method (which I agree with). But in some of these tests I want to force the create or blob deploy method.